### PR TITLE
IGNITE-6373 Create example for local and distributed k-means algorithm

### PR DESCRIPTION
**Pull request by oignatenko**
- examples implemented
-- verified with diffs overview, clean build and execution of examples

### CODE REVIEW

File: modules/ml/src/test/java/org/apache/ignite/ml/clustering/KMeansUtil.java [hunk 1]
```diff
@@ -22,9 +22,9 @@
 import static org.junit.Assert.assertTrue;
 
 /** Base test for k-means algorithms. */
```
Comment (ybabak) [inline modules/ml/src/test/java/org/apache/ignite/ml/clustering/KMeansUtil.java]: wrong javadoc

File: modules/ml/src/test/java/org/apache/ignite/ml/clustering/KMeansDistributedClustererTestSingleNode.java [hunk 2]
```diff
@@ -40,7 +40,6 @@
 import org.apache.ignite.ml.math.impls.vector.DenseLocalOnHeapVector;
 import org.apache.ignite.testframework.junits.common.GridCommonAbstractTest;
 import org.junit.Assert;
-import org.junit.Test;
 
 import static org.apache.ignite.ml.clustering.KMeansUtil.checkIsInEpsilonNeighbourhood;
```
Comment (ybabak) [inline modules/ml/src/test/java/org/apache/ignite/ml/clustering/KMeansDistributedClustererTestSingleNode.java]: Please add some javadoc

Comment (oignatenko) [reply modules/ml/src/test/java/org/apache/ignite/ml/clustering/KMeansDistributedClustererTestSingleNode.java]: added javadoc as suggested

File: modules/ml/src/test/java/org/apache/ignite/ml/clustering/KMeansUtil.java (hunk 1 above)
Comment (oignatenko) [reply modules/ml/src/test/java/org/apache/ignite/ml/clustering/KMeansUtil.java]: corrected javadoc, thanks for catching

**Pull request closed without merging**